src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -29,10 +25,11 @@ internal abstract partial class MemberSemanticModel : CSharpSemanticModel
     {
         private readonly Symbol _memberSymbol;
         private readonly CSharpSyntaxNode _root;
-        private readonly DiagnosticBag _ignoredDiagnostics = new DiagnosticBag();
-        private readonly ReaderWriterLockSlim _nodeMapLock = new ReaderWriterLockSlim(LockRecursionPolicy.NoRecursion);
+        private readonly DiagnosticBag _ignoredDiagnostics = new();
```

Comment:
>new() [](start = 61, length = 5)

I do not believe this is a change for the better. One can no longer use a simple textual search for "new DiagnosticBag" to find all places allocating new bag, for example. IDE is not always there and when it is available, FindAllReferences can be extremely slow, especially when IDE just started.